REQUIREMENTS.html

```diff
@@ -55,7 +55,7 @@ <h3>General requirements:</h3>
 <a href="https://gdal.org">https://gdal.org</a>
 </li>
 
-<li><b>Python &gt;= 2.7 (recommended &gt;= 3.5)</b> (for temporal framework, scripts, wxGUI, and ctypes interface)<br>
+<li><b>Python &gt;= 3.5</b> (for temporal framework, scripts, wxGUI, and ctypes interface)<br>
```

Comment:
what is the minimum Python 3.x? really 3.5?

Reply:
That's still to be defined... 

See RFC 8: Language Standards Support: Python (Draft), https://trac.osgeo.org/grass/wiki/RFC/8_LanguageStandardsSupport

Reply:
Since, as the RFC says, _the oldest still-alive Python version is 3.6, to be retired in December 2021_, I don't see any reason to have anything lower than 3.6 here.

Reply:
Updated to Python 3.6 in dcfb6c548